app/views/training_status/show.json.jbuilder

```diff
@@ -2,12 +2,14 @@
 
 json.course do
   json.training_modules @assigned_training_modules do |training_module|
-    training_progress_manager = TrainingProgressManager.new(@user, training_module)
+    tp_manager = TrainingProgressManager.new(@user, training_module)
```

Comment:
I guess this was just to satify rubocop with the long `completion_time` line? That shouldn't be a problem if the calculation gets moved to a `TrainingProgressManager` instance method.

Reply:
Yes. Okay I'll move it there.

Reply:
Please change it back to `training_progress_manager` as well.

Reply:
@shashwatkathuria ping

Reply:
Done! @ragesoss